### PR TITLE
docs(README): fix broken logo link

### PR DESCRIPTION
**Pull request by xMohamd**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [x] Other... Please describe:  fix broken logo link in README

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <h1 align="center">Angular - The modern web developer's platform</h1>
 
 <p align="center">
-  <img src="aio/src/assets/images/logos/angular/angular_renaissance.png" alt="angular-logo" width="120px" height="120px"/>
+  <img src="docs/images/angular_gradient.png" alt="angular-logo" width="120px" height="120px"/>
```
Comment (JeanMeche) [inline README.md:4]: we could probably use the gif we have on adev : `adev/src/assets/images/press-kit/angular_icon_gradient.gif`

Comment (xMohamd) [reply README.md]: @JeanMeche Done.

**Comment by JeanMeche** [comment]
Can you squash your commits so the change is a single commit ?

**Comment by xMohamd** [comment]
@JeanMeche  Check it

**Review by sonukapoor — APPROVED** [review]
Thanks for providing this fix 👍

**Review by josephperrott — APPROVED** [review]
LGTM

**Review by josephperrott — APPROVED** [review]
LGTM

**Comment by atscott** [comment]
This PR was merged into the repository by commit 13d37e0b4380a8aedddc62b0e07749e3c0d202d3.

**Pull request closed without merging**